packages/wiz-ui-react/src/components/base/inputs/search-input/components/search-input.tsx

```diff
@@ -102,7 +102,7 @@ const SearchInput: FC<Props> = ({
           onClose={() => setIsPopupOpen(false)}
           isDirectionFixed={isDirectionFixed}
         >
-          <WizHStack wrap={false}>
+          <WizHStack nowrap={true}>
```

Comment:
全般的にですが、`={true}` の Props は省略して書けます。

```suggestion
          <WizHStack nowrap>
```